.env.default

```diff
@@ -11,6 +11,9 @@ HTTP_SERVER_PORT=3000
 HTTP_SERVER_HOST=0.0.0.0
 NATS_URL=localhost:4222
 LAMBDAS_URL=
+LIGHTHOUSE_URL=
```

Comment:
what's this used for? Should it be able to reach an outside lighthouse server?

Reply:
no, it's all internal, for the status check, I'll map them in catalyst-owner

Reply:
can we use the same name for the env var then?